lib/engine-jit/src/serialize.rs

```diff
@@ -37,9 +39,58 @@ pub struct SerializableCompilation {
 
 /// Serializable struct that is able to serialize from and to
 /// a `JITArtifactInfo`.
-#[derive(Serialize, Deserialize)]
+#[derive(Serialize, Deserialize, BorshSerialize, BorshDeserialize)]
```

Comment:
`#[cfg_attr(feature = "enable-borsh", derive(BorshSerialize, BorshDeserialize))]`?

Reply:
This is not under feature, because Serialize and Deserialize was always enabled, whereas crate like lib/compiler/Cargo.toml has optional serde, thus also optional borsh